modules/backend/assets/js/october.tabformexpandcontrols.js

```diff
@@ -160,4 +160,209 @@
+                 $activated.prev().children('.primary-tabss a').click().focus();
+             }
+             event.preventDefault();
+         }
+         // strike down or right in the tab
+         if (event.keyCode == 40 || event.keyCode == 39) {
+             // find next tab
+             // if we are on last => activate first
+             $activated = $('.primary-tabs a[aria-selected="true"]').parent();
+             if ($activated.is('.primary-tabs li:last-child')) {
+                 $('.primary-tabs li:first-child a').click().focus();
+             } else {
+                 // else activate next
+                 $activated.next().children('.primary-tabs a').click().focus();
+             }
+             event.preventDefault();
+         }
+     });
+
+     /* Secondary Tabs */
+
+     // if no selected => select first
+     if ($('.secondary-tabs a[aria-selected]').length == 0) {
+         $('.secondary-tabs a:first').attr('aria-selected', 'true');
+         $('.secondary-tabs a:first').attr('tabindex', '0');
+     }
+
+     /* click on a link */
+     $('.secondary-tabs a').click(function() {
+         // remove aria selected on all links + remove focusable
+         $('.secondary-tabs a').attr('aria-selected', 'false');
+
+         // add aria selected on $this + focusable
+         $('.secondary-tabs a.focus-visible').attr('aria-selected', 'true');
+
+         // Important - Must be set to true for October to set class="active" in the <li>
+         return true;
+     });
+
+     $('body').on('keydown','.secondary-tabs', function(event) {
+         // strike up or left in the tab
+         if (event.keyCode == 37 || event.keyCode == 38) {
+             // find previous tab
+             // if we are on first => activate last
+             $activated = $('.secondary-tabs a[aria-selected="true"]').parent();
+             if ($activated.is('.secondary-tabs li:first-child')) {
+                 $('.secondary-tabs li:last-child a').click().focus();
+             } else {
+                 // else activate previous
+                 $activated.prev().children('.secondary-tabs a').click().focus();
+             }
+             event.preventDefault();
+         }
+         // strike down or right in the tab
+         if (event.keyCode == 40 || event.keyCode == 39) {
+             // find next tab
+             // if we are on last => activate first
+             $activated = $('.secondary-tabs a[aria-selected="true"]').parent();
+             if ($activated.is('.secondary-tabs li:last-child')) {
+                 $('.secondary-tabs li:first-child a').click().focus();
+             } else {
+                 // else activate next
+                 $activated.next().children('.secondary-tabs a').click().focus();
+             }
+             event.preventDefault();
+         }
+     });
+
+     /* Content Tabs */
+
+     // if no selected => select first
+     if ($('.content-tabs a[aria-selected]').length == 0) {
```

Comment:
@ayumihamsaki is there a reason you're not using strict compare `===` || `!==`?

Reply:
Fixed.